auth/src/lib.rs

```diff
@@ -67,5 +68,21 @@ pub async fn pgpool_init(db_uri: &str) -> io::Result<PgPool> {
         .await
         .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
 
+    // Post-migration logic for 0003.
+    // This is done here to skip the need for postgres extensions.
+    let names: Vec<(String,)> =
+        sqlx::query_as("SELECT account_name FROM users WHERE user_id IS NULL")
+            .fetch_all(&pool)
+            .await
+            .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
+    for (name,) in names {
+        sqlx::query("UPDATE users SET user_id = $1 WHERE account_name = $2")
+            .bind(User::new_user_id())
+            .bind(name)
+            .execute(&pool)
+            .await
+            .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
+    }
```

Comment:
What's the motivation for avoiding a Postgres extension? It seems we are adding a bit of complexity here, as well as running this query on every startup, rather than just once, which it would if it was done in a migration.

Reply:
Oh, and doing it like this means we can't make the column NOT NULL, even though it technically is. It feels a bit awkward to me.

Reply:
Since the IDs will fundamentally be strings, and will always be generated in our application, having a Ulid postgres extension is extra work with low benefit IMO.
Using the sqlite ulid extension led to the project id bug due to being able to generate an id of the correct format in two places https://github.com/shuttle-hq/shuttle/pull/1387. Also, that extension made them lowercase, which also added work #1424

Reply:
> What's the motivation for avoiding a Postgres extension? It seems we are adding a bit of complexity here, as well as running this query on every startup, rather than just once, which it would if it was done in a migration.

RDS doesn't support ULID extension, so that's kind of a no-go. 

A better solution in my opinion though would be to use something like that: https://github.com/geckoboard/pgulid/blob/master/pgulid.sql (Apache 2.0, I think we can use it ?), and generate IDs in Postgres.

Reply:
I like the suggestion by Jocelyn. Maybe we can use it instead?

Reply:
Call notes: we are going to remove this soon - ie this is only temp. So this will be fine for now.